### ISSUE TITLE
Request: hide other cursor shapes instead of only default shape

### ISSUE THREAD

**Issue by Sneezor** [Codeusa/Borderless-Gaming#130]
I don't know if this has been addressed or asked already, I noticed using the hide cursor function only hides the default point but not the other othes, e.g. vertical resize.

Screenshot with example: http://prntscr.com/7kqdda

I guess only a few of those pointers are hidden but hiding every single type of pointer would be a good idea.

**Comment by psouza4** [comment]
We'll have to find out how prevalent an issue this is for people to determine priority.  There's some inherent risk in hiding the cursor in case of application crash or unexpected termination that is not as risky with the default cursor (since it changes so frequently).  Worst case scenario is restarting the computer.  I think we should wait and see if other people even want this one first.

**Comment by psouza4** [comment]
I haven't seen any additional interest for your enhancement request.  It sounds like maybe you want a different program that targets streamer and presentation needs.

**Comment by Sneezor** [comment]
It’s not that I have no interest about that, but rather a non-pressing matter. There are just a few scenarios where a pointer aside from the default one is displayed, but I thought you might want to address this.

 

At first glance it looks rather simple, I could imagine there might be a collection of pointers/attributes for the current pointer that you can iterate through and set to hidden instead of the current implementation of whichever method handles that, but it’s probably harder to do than it looks and the issue doesn’t really matter that much.

 

TL;DR: You might want to know that other pointer types aren’t hidden and add it to the end of your task list to fix later on.

**Comment by psouza4** [comment]
I said 'additional' interest, as in from other people who would also like this feature.  Nobody else has chimed in.

But yes, it's not a simple process to alter global cursors, especially if you want to retain them if an application dies or closes unexpectedly.  There's a huge danger of permanently losing the mouse cursor until Windows has been restarted and it becomes messy quickly.  It just doesn't seem worthwhile for a tiny use case.
